smarts/core/remote_agent_buffer.py

```diff
@@ -92,12 +92,7 @@ def __init__(self, zoo_manager_addrs=None, buffer_size=3):
             self._remote_agent_future() for _ in range(self._buffer_size)
         ]
 
-        atexit.register(self.destroy)
-
     def destroy(self):
```

Comment:
Does anything call `destroy()` now?  (I don't see any calls to it.)

If not, then will the threads be terminated in the "normal" (non-crashing) case?  Alternatively, if it's not called and definitely not needed, maybe we should remove the method?

Reply:
`RemoteAgentBuffer()` is created and managed by `AgentManager()`. 

`RemoteAgentBuffer.destroy()` is called by `AgentManager.destroy()`. 

https://github.com/huawei-noah/SMARTS/blob/7c60c80ad74add4d6c2d8fec5629a23026cb1cb6/smarts/core/agent_manager.py#L76-L78 .